src/Shared/MemberInfoExtensions.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
Omit per #674

Reply:
Done, although I'm not too sure about the license status of these files which we copy-paste into the provider... Npgsql is licensed under the PostgreSQL license, which isn't the same as the Apache license which EF Core uses.

@ajcvickers, @divega anything to say here??